app/models/form.rb

```diff
@@ -5,15 +5,15 @@ class Form
 
   def initialize(form_path, start_year = "", sections_in_form = [], type = "lettings")
     if type == "sales"
+      @start_date = Time.zone.local(start_year, 4, 1)
+      @end_date = Time.zone.local(start_year + 1, 7, 1)
```

Comment:
Added this to the top, because we pass in `self` as a form to the sections when initializing them, and we would want the start and end date to be set before we do that